### PR TITLE
Fix #1

### LINKED ISSUE #1: Bootstrap frontend
- Initialise the front-end with Create React App and Typescript.

### PR DESCRIPTION
**Pull request by jnanatekumudi**

### CODE REVIEW

File: Frontend/my-app/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```
Comment (onprem) [inline Frontend/my-app/README.md]: Let's replace the README with some relevant content. For now, something like
```md
# Project Name
description...
```
should be fine.

File: Frontend/my-app/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```
Comment (onprem) [inline Frontend/my-app/public/index.html]: Replace this description with something more relevant.

File: Frontend/my-app/public/index.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    <!--
+      Notice the use of %PUBLIC_URL% in the tags above.
+      It will be replaced with the URL of the `public` folder during the build.
+      Only files inside the `public` folder can be referenced from the HTML.
+
+      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+      work correctly both with client-side routing and a non-root public URL.
+      Learn how to configure a non-root public URL by running `npm run build`.
+    -->
+    <title>React App</title>
```
Comment (onprem) [inline Frontend/my-app/public/index.html]: And the title too.

File: Frontend/my-app/public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (onprem) [inline Frontend/my-app/public/manifest.json]: Same with this file.

**Review by onprem — COMMENTED** [review]
Everything looks good, but I have a few suggestions. The directory structure is currently `Frontend/my-app/...everything...`, the `my-app` directory is redundant, let's move to `frontend/...everything...`.

**Comment by jnanatekumudi** [comment]
Made all the necessary changes to the files mentioned.

**Comment by onprem** [comment]
We still have the old `my-app` directory and a few files in it, let's remove that completely.

**Comment by jnanatekumudi** [comment]
> We still have the old `my-app` directory and a few files in it, let's remove that completely.

Done. I forgot to stage those changes and it did not get pushed previously.

**Comment by onprem** [comment]
You pushed `node_modules` :smile:

**Review by onprem — APPROVED** [review]
LGTM!

**Pull request merged**